docs/OCP_CI_Tutorials/Scenarios/Reporting_Guide.md

```diff
@@ -0,0 +1,128 @@
+
+- [TestGrid Homepage](https://testgrid.k8s.io/)
+- [TestGrid Documentation and Source Code](https://github.com/kubernetes/test-infra/tree/master/testgrid)
+
+### TestGrid Configuration Example and Description
+
+`config/testgrids/openshift/redhat-openshift-4.12-interop.yaml`: 
+
+```yaml
+dashboards:
+- name: redhat-openshift-4.12-interop
+  dashboard_tab:
+  - name: quay-3.7
+    test_group_name: quay-3.7-openshift-4.12-interop
+    open_test_template:
+      url: https://prow.ci.openshift.org/view/gs/<gcs_prefix>/<changelist>
+    results_url_template:
+      url: https://prow.ci.openshift.org/job-history/<gcs_prefix>
+
+test_groups:
+- name: quay-3.7-openshift-4.12-interop
+  gcs_prefix: origin-ci-test/logs/periodic-ci-rhpit-interop-tests-master-ocp-412-quay37-interop-quay-tests-aws-ipi-ocp412
+  days_of_results: 60
+```
+
+- `dashboards`: A YAML stanza to describe a TestGrid dashboard.
+  - `name`: The name of the dashboard. Please notice this name is the same as the filename.
+  - `dashboard_tab`: A YAML stanza defining a list of "tabs" in a dashboard. Each of these items can be thought of as a single scenario.
+    - `name`: The name of the "tab" or test. This should be more readable than the long Prow job name. In this example I am using "quay-3.7" rather than the long Prow job name defined later in the configuration.
+    - `test_group_name`: Referring to the test group set up for a specific Prow job. See the `test_groups` stanza for the list of test groups.
+    - `open_test_template`: Template for the URL linking to a specific test.
+      - `url`: URL template. This should not change for our purposes.
+    - `results_url_template`: Template for the URL linking to specific test results.
+      - `url`: URL template. This should not change for our purposes.
+
+- `test_groups`: A list of Prow jobs to be able to use in your dashboard.
+  - `name`: Essentially an alias for the Prow job name. We use this name in the `test_group_name` key above. This value needs to be unique to not only this document, but other `test_groups` items in other config files.
+  - `gcs_prefix`: The Google Cloud Storage prefix path for the Prow results. Should always be `origin-ci-test/logs/` plus the name of the Prow job.
+  - `days_of_results`: How many days of results should be seen on TestGrid. I haven't seen this value go above 60 days, but I also haven't tried going any higher than that.
+
+### How to Create a New Dashboard
+
+Creating a new TestGrid dashboard is very simple. Before you get started you will need to [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository into your personal GitHub organization.
+
+
+Once you have your fork created, follow these instructions to create a new dashboard:
+
+1. Prepare Git:
+   1.  Clone a copy of your fork to your computer
```

Comment:
```suggestion
   1.  Clone a copy of your fork to your computer.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)